lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/test/java/io/lighty/applications/rnc/module/RncLightyModuleSmokeTest.java

```diff
@@ -10,30 +10,120 @@
 import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.when;
 import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+import static org.testng.AssertJUnit.assertEquals;
 
 import io.lighty.applications.rnc.module.config.RncLightyModuleConfigUtils;
-import io.lighty.applications.rnc.module.config.RncLightyModuleConfiguration;
 import io.lighty.core.controller.impl.config.ConfigurationException;
+import java.net.Socket;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.nio.file.Paths;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
+import java.time.Duration;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509ExtendedTrustManager;
+import org.eclipse.jetty.http.HttpStatus;
 import org.testng.annotations.Test;
 
 public class RncLightyModuleSmokeTest {
+
     private static final int MODULE_TIMEOUT = 60;
+    private static final String HTTPS_URI = "https://127.0.0.1:8888";
+    private static final String HTTP_URI = "http://127.0.0.1:8888";
+    private static final String TOPOLOGY_PATH = "/restconf/data/network-topology:network-topology";
 
     @Test
-    public void rncLightyModuleSmokeTest() throws ConfigurationException {
-        final RncLightyModule rncModule = new RncLightyModule(RncLightyModuleConfigUtils.loadDefaultConfig(),
-                MODULE_TIMEOUT);
-        rncModule.initModules();
-        rncModule.close();
+    public void rncLightyModuleDefaultConfigTest() throws Exception {
+        final var rncModule = new RncLightyModule(RncLightyModuleConfigUtils.loadDefaultConfig(), MODULE_TIMEOUT);
+        assertTrue(rncModule.initModules());
+        final var http1Client = getHttp1Client();
```

Comment:
Unnecessary variable `http1Client`.

Reply:
I like it and I would like to keep this variable as it describes that this test use HTTP1 client